README_CN.md

```diff
@@ -0,0 +1,222 @@
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/jenkins-x/jx-docs)
```

Comment:
hi，i want to know what font is appropriate of the English language link

Reply:
Would you be more specific? I didn't get your point.

Reply:
oh，sorry，i try to give you some example 
```
# [[中文](https://github.com/jenkins-x/jx-docs/pull/README_CN.md)](./README_CN.md)

# Jenkins X Docs
```
```
## [[中文](https://github.com/jenkins-x/jx-docs/pull/README_CN.md)](./README_CN.md)

# Jenkins X Docs
```
```
### [[中文](https://github.com/jenkins-x/jx-docs/pull/README_CN.md)](./README_CN.md)

# Jenkins X Docs
```
please in three examples choice a right size of `中文`

Reply:
MIO, it would be great if we have the link. The size does not matter. This is not a blocking issue.

```
[中文](./README_CN.md) [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/jenkins-x/jx-docs)
```